src/services/cache_service.ts

```diff
@@ -0,0 +1,22 @@
+class CacheService {
```

Comment:
As written, it's not clear to me why this needs to be a class. I think I made a similar comment regarding the TableService.  This class only contains static methods (although you don't declare them as such), which means you're using the class for namespace purposes only, which usually ends up being unnecessary.

Reply:
I will probably be removing the `TableService` in a future pr so not going to resolve that here. I changed `CacheService` to be a `const object` instead of a class.